cobc/pplex.l

```diff
@@ -2794,6 +2813,12 @@ pp_text_list_add (struct cb_text_list *list, const char *text,
 	struct cb_text_list	*p;
 	void			*tp;
 
+	for (p = list; p; p = p->next) {
+		if (!strncmp(text, p->text, size + 1)) {
```

Comment:
Why using not use strcmp here?

Reply:
Just a habit of using `strncmp`. Changed it.